fire/cli/info.py

```diff
@@ -743,10 +771,16 @@ def sag(sagsid: str, **kwargs):
 
         return
 
-    sager = fire.cli.firedb.hent_alle_sager()
+    if not sager:
+        sager = fire.cli.firedb.hent_alle_sager()
+
     fire.cli.print("Sagsid     Behandler           Beskrivelse", bold=True)
     fire.cli.print("---------  ------------------  -----------")
     for sag in sager:
+        if not sag.aktiv:
+            # Spring inaktive sager over
+            continue
+
```

Comment:
Her kunne man måske give mulighed for også at vise inaktive sager ved at angive et kommandolinjeflag